templates/user/login-form.php

```diff
@@ -39,15 +39,15 @@
 
 				<label for="sensei_user_login"><?php esc_html_e( 'Username or Email', 'sensei-lms' ); ?> </label>
 
-				<input type="text" name="log" id="sensei_user_login" class="input" value="" size="20">
+				<input type="text" name="log" id="sensei_user_login" class="input input-text" value="" size="20">
```

Comment:
I didn't name `input-text` as per the convention because the same class name is used in the registration form. It's also looks like a convention to use this class in form input text fields, also used in woocommerce forms.